apps/blog/src/posts/introducing-manifest-editor/introducing-manifest-editor.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: The Manifest Editor
+excerpt: Introducing a new way to generate and edit your manifest.
+description: Introducing a new way to generate and edit your manifest.
+date: 2022-11-03
+updatedDate: 2022-11-03
+trending: true
+featured: true
+image: posts/introducing-manifest-editor/manifest-editor.png
+isPost: true
+backUrl: '/'
+author:
+  name: Jaylyn Barbee
+  title: PWABuilder SWE
+tags:
+  - post
+  - PWA
+  - manifest
+  - pwabuilder.com
+---
+
+With our site redesign that was released in early October, some of you may have noticed our new manifest editor component! We wanted a guided experience that would make this daunting process much easier for our user. Being that its a component, it is also something that we will be able to reuse across our toolings when the time is right. We will continue to improve this experience, adding more information about each field even step by step instructions on how to implement them. The best part is, all of this will happen without even having to leave pwabuilder.com!
```

Comment:
```suggestion
With our site redesign that was released in early October, some of you may have noticed the new manifest editor component! We wanted a guided experience that would make this daunting process much easier for our users. Being that it is a component, we will be able to reuse it across PWABuilder toolings when the time is right. We will continue to improve this experience, add more information about each field, and provide step by step instructions on how to implement them. The best part is, all of this will happen without having to leave pwabuilder.com!
```